news_items/034_matching_element/index.html

```diff
@@ -0,0 +1,65 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+    PrairieLearn now has a new <code class="user-output">pl-matching</code> element, thanks to the efforts of 
+    <a href="https://github.com/nealterrell">Neal Terrell</a> (California State University, Long Beach), with input from
+    <a href="https://github.com/jonatanschroeder">Jonatan Schroeder</a>.
+</p>
+
+<p>
+    This element can be used to create two-column &quot;matching&quot; problems, where students
+    must select an option that best matches each entry in a list of statements.
+</p>
+
+<img class="border rounded" src="pl-matching.png" width="600px">
```

Comment:
I'm seeing a double-border on the left of this image in the news item:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/875833/131425883-390a4dff-a49b-4246-849f-955ef6372f61.png">

Reply:
```suggestion
<img src="pl-matching.png" width="600px">
```